cli/ops/fs.rs

```diff
@@ -6,7 +6,6 @@ use crate::op_error::OpError;
 use crate::ops::dispatch_json::JsonResult;
 use crate::state::State;
 use deno_core::*;
-use remove_dir_all::remove_dir_all;
```

Comment:
I think this import can be removed (along with entry in `Cargo.toml`) and replaced with [`std::fs::remove_dir_all`](https://doc.rust-lang.org/std/fs/fn.remove_dir_all.html)

Reply:
Will do. Probably this is a good opportunity to start shifting over to the tokio::fs versions, as you propose in #4188.

Q: after removing the import from `Cargo.toml`, should I also commit an updated `Cargo.lock`? Or do you defer that until just before a release?

Reply:
Yes, but #4188 should be done in a separate PR (I think I saw this code in one of your PRs)

Yes, Cargo.lock should be committed as well

Reply:
Ok, there's a problem using `std::fs::remove_dir_all` as I do in `97c1a69` (below). Then the tests fail on Windows with a `error: Uncaught Error: The directory is not empty. (os error 145)`. I assume this comes from `removeAllSyncDirSuccess` or `removeAllDirSuccess`, but those tests aren't currently written so as to catch that failure. So for the moment, I guess we should stick with `remove_dir_all::remove_dir_all`? It may be that the `tokio::fs` version works better on Windows.

Reply:
Okay, makes sense 👍

Reply:
@dubiousjim I don't understand - these tests currently are fine?

I really don't want to add new crate dependencies for functionality that is in std.